docs/articles/documentation/test-api/actions/press-key.md

```diff
@@ -23,22 +23,63 @@ Key Type                   | Example
 -------------------------- | ------
 Alphanumeric keys          | `'a'`, `'A'`, `'1'`
 Modifier keys              | `'shift'`, `'alt'` (⌥ key on macOS), `'ctrl'`, `'meta'` (*meta* key on Linux and ⌘ key on macOS)
-Navigation and action keys | `'backspace'`, `'tab'`, `'enter'`, `'capslock'`, `'esc'`, `'space'`, `'pageup'`, `'pagedown'`, `'end'`, `'home'`, `'left'`, `'right'`, `'down'`, `'ins'`, `'delete'`
+Navigation and action keys | `'backspace'`, `'tab'`, `'enter'`
 Key combinations           | `'shift+a'`, `'ctrl+d'`
 Sequential key presses     | Any of the above in a space-separated string, e.g. `'a ctrl+b'`
 
-In addition to key presses handled by webpage elements, the `t.pressKey` action also allows you to execute certain key presses processed by the browser.
+The following navigation and action keys are supported:
 
-* `'ctrl+a'`, `'backspace'`, `'delete'`, `'left'`, `'right'`, `'up'`, `'down'`, `'home'`, `'end'`, `'enter'`, `'tab'`, `'shift+tab'`, `'shift+left'`, `'shift+right'`, `'shift+up'`, `'shift+down'`, `'shift+home'`, `'shift+end'`
+* `'backspace'`
+* `'tab'`
+* `'enter'`
+* `'capslock'`
+* `'esc'`
+* `'space'`
+* `'pageup'`
+* `'pagedown'`
+* `'end'`
+* `'home'`
+* `'left'`
+* `'right'`
+* `'up'`
+* `'down'`
+* `'ins'`
+* `'delete'`
 
-With the exception of the keys and combinations listed above, the `t.pressKey` action will not invoke integrated browser keystrokes.
+When an end-user normally presses a key or key combination, the browser dispatches an event that triggers handlers in page code. Most browsers also respond to common keystrokes with their integrated actions (for instance, they select text when you press `Cmd/Ctrl+A` or copy objects to the clipboard when you press `Cmd/Ctrl+C`).
 
-For elements with the `contentEditable` attribute, the following key presses are supported.
+The `t.pressKey` action triggers only page handlers for most keystrokes. However, the following keys and key combinations additionally invoke browser processing:
```

Comment:
Can we rephrase this sentence? It's hard to understand what we want to say. 
E.g. those keystrokes should ivnoke browser processing, but testcafe doesn't or we additionally emulate browser behavior too?

Reply:
TestCafe really emulates browser's behavior but not for all elements:

Shortcut | Elements
--- | ---
ctrl+a | 'editable' elenents
backspace | 'editable' elenents
del | 'editable' elenents
left | 'editable' elenents, select element, radio buttons
right | 'editable' elenents, select element, radio buttons
up |inputs and textareas, select element, radio buttons
down | inputs and textareas, select element, radio buttons
shift+left | inputs and textareas
shift+right | inputs and textareas
shift+up | inputs and textareas
shift+down | inputs and textareas
shift+home | inputs and textareas
shift+end | inputs and textareas
home | inputs and textareas
end | inputs and textareas
enter | select element, inputs and textareas, links
tab | focus next focusable element on the page
shift+tab | focus previous focusable element on the page
esc | select element

*'Editable' elenents: input, textarea, conteneditable.

Maybe we need more information about behavior we emulates (e.g. for `tab` pressing we close select element's popup if it's open,  move focus to the next focusable element on the page. If the last focusable element on the page is selected, the first element will be selected. If the new focusable element is editable, we'll select all its content)

Reply:
Well, let's clarify this behaviour then.

Reply:
Sure, okay.

BTW, `tab` descriptions seems an overkill, I think we can go without it for now.

Please, clarify element types in the table.

If `'editable' elements` include all inputs, how does `'editable' elements` differ from `'editable' elements, radio buttons`?

Reply:
We consider editable elements in which typing can be performed:
- inputs with certain [types](https://github.com/DevExpress/testcafe-hammerhead/blob/8c832c78e04ec1d69a9da5af629629819a60a224/src/client/utils/dom.ts#L31)
- textareas
- elements with contenteditable attribute